alias/.rpdk-config

```diff
@@ -1,4 +1,5 @@
 {
+    "artifact_type": "RESOURCE",
```

Comment:
What's the purpose of the changes in this file?

Reply:
This file is generated by the CFN java plugin. The update here was the result of an update to the plugin.